src/service.js

```diff
@@ -157,14 +167,25 @@
                 reload: function (widgetId) {
                     validateRecaptchaInstance();
 
-                    // $log.info('Reloading captcha');
                     recaptcha.reset(widgetId);
 
                     // Let everyone know this widget has been reset.
                     $rootScope.$broadcast('reCaptchaReset', widgetId);
                 },
 
                 /**
+                 * Set reCaptcha language
+                 */
+                setLang: function (widgetId, lang) {
+                    var cli = window.___grecaptcha_cfg.clients[widgetId];
```

Comment:
things that start with `_` and especially `__` and even more so `___` are internal methods and should not be relied upon by third parties are they can change or disappear. `window.grecaptcha` is the official api.
Furthermore, variable names like `W` and `wk` should not be replied upon because those are generated by the build process and will change build to build, version to version.